loki/values.yaml

```diff
@@ -0,0 +1,2 @@
+promtail:
```

Comment:
at least persistence by pvc should get enabled as it is off by default

Reply:
PVC enabled by default